AwsCryptographicMaterialProviders/codegen-patches/AwsCryptographicMaterialProviders/java/dafny-4.9.0.patch

```diff
@@ -0,0 +1,29 @@
+diff --git b/AwsCryptographicMaterialProviders/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/materialproviders/model/CollectionOfErrors.java a/AwsCryptographicMaterialProviders/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/materialproviders/model/CollectionOfErrors.java
+index 796adaa3e..d1eea14c4 100644
+--- b/AwsCryptographicMaterialProviders/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/materialproviders/model/CollectionOfErrors.java
++++ a/AwsCryptographicMaterialProviders/runtimes/java/src/main/smithy-generated/software/amazon/cryptography/materialproviders/model/CollectionOfErrors.java
+@@ -4,6 +4,9 @@
+ package software.amazon.cryptography.materialproviders.model;
+ 
+ import java.util.List;
++import java.util.stream.Collectors;
++import software.amazon.awssdk.utils.CollectionUtils;
++import software.amazon.awssdk.utils.StringUtils;
```

Comment:
`CollectionUtils` and `StringUtils` seems to be unused.

Reply:
Sorry. Missed this in my 1st review.

Reply:
Hmm... that must have been from an earlier attempt on my part.
All is groovy.
I put my revision script in the last commit message!